pkg/machines/libvirt-dbus.js

```diff
@@ -379,11 +379,15 @@ LIBVIRT_DBUS_PROVIDER = {
                     });
         }
 
-        if (options.destroy) {
+        if (options.destroy)
             return destroy().then(undefine());
-        } else {
-            return undefine();
-        }
+        else
+            return undefine()
```

Comment:
Style nitpick: For these multi-line then/else clauses we still use {} in both, for better readability.